libraries/ui-library/src/components/six-header/six-header.tsx

```diff
@@ -56,6 +56,9 @@ export class SixHeader {
   /** Indicates if content should be shifted down when search field is visible. */
   @Prop() shiftContent = false;
 
+  /** Set whether the hamburger menu should be visible or not */
+  @Prop() showHamburgerMenu = true;
```

Comment:
Recently learned, that is it better to set the default value of a property to `false`, so that you can write

`<six-header hide-hambumber-menu>` instead of `<six-header show-hambumber-menu="false">`

so we might want to change this to:

`@Prop() hideHamburgerMenu = false;`

Also see: https://github.com/stencil-community/stencil-eslint/blob/main/docs/ban-default-true.md

Reply:
makes sense! i updated the code.